esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -116,6 +116,50 @@ IPropertyTree * fetchConfigInfo(const char * config,
     return methodstree;
 }
 
+IPropertyTree * fetchConfigInfo(const char * config,
+                                const char* bindingId)
+{
+    IPropertyTree * methodstree = NULL;
+
+    if (!config || !*config)
+    {
+        throw MakeStringException(-1,"Empty config detected");
+    }
+    else
+    {
+        Owned<IPropertyTree>  configTree = createPTreeFromXMLString(config, ipt_caseInsensitive);
+        //Now let's figure out the structure of the configuration passed in...
+
+        StringBuffer rootname;
+        configTree->getName(rootname);
+
+        IPropertyTree * deftree = NULL;
```

Comment:
picky: Moving the variables nearer to where they are used would make this easier to read/understand.  E.g., line 143.